packages/react-charting/src/utilities/utilities.ts

```diff
@@ -502,6 +502,7 @@ export function calloutData(values: (ILineChartPoints & { index?: number })[]) {
       legend: string;
       y: number;
       color: string;
+      xAxisCalloutData?: string;
```

Comment:
Could you add an example also showing how to use this property.
The examples are in packages/react-examples/src/react-charting/.

You can add it to LineChart/LineChart.CustomAccessibility example

Reply:
Sure... So an example depicting use of a custom callout containing xAxisCallout as well...

Reply:
The example is mentioned in the codepen link attached in the PR description